gbfs.md

```diff
@@ -501,7 +564,7 @@ Example:
 ```
 
 ### system_hours.json
-Describes the system hours of operation.
+This optional file is used to describe hours and days of operation when vehicles are available for rental. If `system_hours.json` is not published it indicates that vehicles are available for rental 24 hours a day, 7 days a week.
```

Comment:
I think more clarification is needed that: if a system is outside of hours of operation, it should have no vehicles available in the realtime feeds (i.e. it should set num_bikes_available=0 and have no vehicles in free_bike_status.json). I'm not sure that is mentioned anywhere. The last sentence here makes it sound like consumers need to ingest system_hours just to determine if current real-time availability is outside of hours of operation.

Reply:
> if a system is outside of hours of operation, it should have no vehicles available in the realtime feeds (i.e. it should set num_bikes_available=0 and have no vehicles in free_bike_status.json).

The definition of `num_bikes_available` was changed (discussion in #94, PR #195 )to say just the opposite of this. During off hours `num_bikes_available` now remains constant so someone trying to plan a trip at 6:45 can see how many vehicles are going to be available at 7:00 when the system opens. Same should be true for `free_bike_status`. In a station based system the operator could set `is_renting` to `false` in `station_status` during the time when the system is closed  but there's no equivalent in `free_bike_status`  so you would need to ingest `system_hours`.

Reply:
> Would you be able to also update the system_pricing_plans documentation to clarify start/end interval points in pricing? 

`start` and `end` definitions have been updated in pricing

Reply:
I see. I don't like that the realtime part of the spec does not reflect the realtime rental availability and consumers have to double-check with another mostly static feed, but I do respect conversation in #94. Would it make sense to add guidance to is_renting on whether this should be set to false outside of hours of operation? Currently the description seems to point towards it: "temporarily taken out of service".